features/user_can_post.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Create articles
+  As a visitor
+  In order to discuss plants with others 
+  I would like to be able to post to the forum
+
+  Background:
+    Given I visit the site
+    And I click "FORUM"
+    And I click "New Post"
```

Comment:
Hmm... the story implies that it is a logged-in user that creates the post. Yet,  I can't see a user being authenticated

Reply:
same as above